internal/testing/e2e/consts.go

```diff
@@ -18,32 +18,27 @@ var (
 	zygote OM
 
 	// Indices covers all ticketIndices that will be used in the e2e test
-	// Please update #L394 in the Makefile if you need to add new indices to the e2e tests.
-	// TODO: Unify these indices
-	Indices = []string{SkillAttribute, Map1Attribute, Map2Attribute, LevelIndex, DefenseIndex, BoolIndex, StringIndex}
+	// Please update the ticketIndices in the helm chart for in-cluster end-to-end test if you need to add new indices.
+	Indices = []string{FloatRangeIndex1, FloatRangeIndex2, FloatRangeIndex3, BoolIndex1, StringIndex1}
```

Comment:
Since we want to test atleast one of each type, can make them a little more meaningful? 
MMR --> Float
Role --> String
Demo --> Bool